### PR TITLE
[TRIVIAL] Fixed driver OpenAPI /reveal request definition

### PR DESCRIPTION
**Pull request by mstrug**
# Description
OpenAPI driver `/reveal` request has 3 fields: `solutionId`, `submissionDeadlineLatestBlock`, `auctionId` which is inconsistent with the source code, where `/reveal` request has only 2 fields: `solutionId` and `auctionId`: [source](https://github.com/cowprotocol/services/blob/main/crates/driver/src/infra/api/routes/reveal/dto/solution.rs).

# Changes
`/settle` and `/reveal` endpoints have same API definition: `Solution`, so I've created new API component: `SolutionWithDeadline` and assigned it to `/settle` endpoint. From `Solution` I've removed `submissionDeadlineLatestBlock` field which now aligns with `/reveal` endpoint source code.
Renamed driver settle dto module `solution.rs` to `solution_with_deadline.rs` and name of the struct `Solution` to `SolutionWithDeadline`.

### CODE REVIEW

File: crates/driver/openapi.yml [hunk 1]
```diff
@@ -526,7 +526,19 @@ components:
               gas:
                 type: integer
     Solution:
-      description: Request to the settle and reveal endpoint.
+      description: Request to the reveal endpoint.
```
Comment (MartinquaXD) [inline crates/driver/openapi.yml:529]:
```suggestion
      description: Request to the `/reveal` endpoint.
```

**Review by MartinquaXD — COMMENTED** [review]
I think in order to avoid renaming everything every time we add or remove a field we should consider renaming files, top level structs and API references to sth like `[reveal_]request` and `[reveal_]response` (same for `/solve` and `/settle`)

**Comment by mstrug** [comment]
> I think in order to avoid renaming everything every time we add or remove a field we should consider renaming files, top level structs and API references to sth like `[reveal_]request` and `[reveal_]response` (same for `/solve` and `/settle`)

Renamed `/reveal`, `/solve` and `/settle` related files and struct names and updated `openapi.yml` file.

**Review by sunce86 — APPROVED** [review]
Tests passing without adjustments in the code show that no breaking changes are introduced.

LGTM

**Review by squadgazzz — APPROVED** [review]
LG

**Comment by squadgazzz** [comment]
Btw, I suppose, this is not a `TRIVIAL` but `EASY` change 🙂

**Pull request merged**